app/v1/models/errors/mtdErrors.scala

```diff
@@ -51,6 +52,14 @@ object BVRError extends MtdError("BUSINESS_ERROR", "Business validation error")
 
 object ServiceUnavailableError extends MtdError("SERVICE_UNAVAILABLE", "Internal server error")
 
+// Liability Errors
```

Comment:
Just a tiny minor thing.  Can we move these down to just above the Custom VAT errors?  When we come to (eventually) reversion VAT and improve the errors they'll be easier to find to delete.